presto/test_presto.py

```diff
@@ -0,0 +1,103 @@
+import random
+import unittest
+
+from parameterized import parameterized
+
+from integration_tests.dataproc_test_case import DataprocTestCase
+
+
+class PrestoTestCase(DataprocTestCase):
+    COMPONENT = 'presto'
+    INIT_ACTION = 'gs://dataproc-initialization-actions/presto/presto.sh'
+
+    def setUp(self):
+        pass
+
+    def verify_instance(self, name, coordinators, workers):
+        schema = "test_{}".format(random.randint(0, 1000))
+        table = "table{}".format(random.randint(0, 1000))
+
+        self.__create_schema(name, schema)
+        self.__create_table(name, table)
+        self.__verify_schema(name, schema)
```

Comment:
Please add a small presto query -- it's important to make sure that scheduling tasks and shuffle both work. Something like word count: `SELECT number, count(1) AS total FROM test_5 GROUP BY number ORDER BY total DESC LIMIT 10`.

I think that means you'll need to add some numbers into the hive table.